packages/light-apps/src/ContextGate/ContextGate.tsx

```diff
@@ -14,13 +15,17 @@ import {
 import { Loading } from '@substrate/ui-components';
 import React from 'react';
 
+// FIXME Use PostMessageProvider once we have an extension
+// https://github.com/paritytech/substrate-light-ui/issues/52
+const wsProvider = new WsProvider('ws://localhost:9944');
```

Comment:
cc YJ: Once you have the PR with PostMessageProvider ready, this line should be changed. It would be something like:
- if we're in an extension popup, use `PostMessageProvider`
- if we're in an electron app, use `WsProvider('ws://localhost:9944')`
- if we're in a browser, just use `WsProvider('${remotenode}:9944')` for now